c2cgeoportal/scaffolds/update/CONST_vars.yaml_tmpl

```diff
@@ -362,15 +365,12 @@ vars:
               display: PhantomJS generic test
         hosts:
         - display: {{package}}
-          url: http://{host}/{instanceid}/wsgi
+          url: '{web_prorocol}://{host}/{instanceid}/wsgi'
           type: main
 
     # What web page is authorized to use the API
     authorized_referers:
-    - "http://{host}/{instanceid}"
-    - "https://{host}/{instanceid}"
-    - "http://{host}:8480/{instanceid}"
-    - "https://{host}:8480/{instanceid}"
+    - '{web_prorocol}://{host}/{instanceid}'
```

Comment:
Usually the application is on http or on https, we add booth to make it working on booth case, now with web_protcol we don't needs two entry for the booth case. it we should have booth we can add the manually :-)